docs/UserGuide.md

```diff
@@ -1481,30 +1527,74 @@ To get a list of `COMMAND_WORD`, do `help` with no arguments
 > Example: help add
 > ```
 
-#### Reset UniCa$h
+#### Reset UniCash
+
 Resets UniCa$h to its default state.
 
 Command: `reset_unicash`
 
-Command Words Accepted: `reset_unicash` (case-insensitive)
+Command Words Accepted: `reset_unicash` only (case-insensitive)
+
+<div class="callout callout-info" markdown="span" style="margin-bottom: 20px;">
+Similar to `Clear Transactions`, this command overwrites all existing transactions in UniCa$h.
+Thus, as an added layer of safety, the command has no short form alternatives.
+It also cannot be followed with any arguments, options, or non-whitespace text.
+</div>
+
+
+<div class="callout callout-important" markdown="span" style="margin-bottom: 20px;">
+This command is subject to the filter applied by `find` or `get_total_expenditure`
```

Comment:
Same here, I believe it doesn't matter if a filter is applied, it overrides the base transaction list

Reply:
I'm not too sure what you mean, it overrides the list yes but doesn't reset the predicate

Reply:
Is the phrasing confusing, maybe I can change it to "The output of this command.."